### PR TITLE
feat: Defaulting to MemoryEngine with temporary client

### PR DESCRIPTION
**Pull request by bennyn**

### CODE REVIEW

File: src/main/Account.ts [hunk 1]
```diff
@@ -39,7 +39,7 @@ export default class Account extends EventEmitter {
   constructor(loginData: LoginData, storeEngine: CRUDEngine = new MemoryEngine('temporary')) {
     super();
     this.loginData = {
-      persist: false,
+      persist: (storeEngine instanceof MemoryEngine) ? false : true,
```
Comment (lipis) [inline src/main/Account.ts]: !(storeEngine instanceof MemoryEngine)

Comment (gregor) [reply src/main/Account.ts]: 👍

**Comment by gregor** [comment]
Doesn't do this the opposite of what the description says though? It defaults to temporary client if MemoryEngine is used.

**Comment by bennyn** [comment]
The `persist` flag should be `false` when a `MemoryEngine` is used.

**Pull request merged**